src/react/deckgl.js

```diff
@@ -79,6 +69,7 @@ export default class DeckGL extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    this._setLayerManagerProps(nextProps);
```

Comment:
Feels a little misleading since `LayerManager` is a traditional JS class and doesn't really have props. Just call it `_updateLayerManager`?